### ISSUE TITLE
Hosting on Subdomain

### ISSUE THREAD

**Issue by PaulDebus** [TailorDev/monod#395]
Hey,

Thanks for that awesome project. I am just starting to get to know it but I already like it. 

I have one question though: how can I host monod on a Subdomain on my server? I have forwarded traffic from example.com/monod/ to localhost:12345 and I get most of the page, but the log says it does not find some parts, since it is using the path without the Subdomain. Am I missing something?

Thanks for your time!

**Comment by willdurand** [comment]
Hi! Thanks for the kind words :)

We have deployed Monod on a subdomain too: https://monod.lelab.tailordev.fr/. In your case, it looks like you are not using a subdomain but rather a Apache alias or directory, which might not be well supported to be honest (I just do not know actually).

**Comment by PaulDebus** [comment]
Yeah, I guess that was poor choice of words. What I mean is, I want to access monod at example.com/monod/ I guess that what I meant was Apache alias

I will look into it, if I can add this myself. I am not well practiced in writing Node, but I have an idea how it could work. 

The Chrome dev console says 
```
raven.js:1015 GET http://jukka.elnath.uberspace.de/documents/monod/ 404 (Not Found)
runtime.js:41 GET http://jukka.elnath.uberspace.de/appcache/manifest.html 404 (Not Found)
```
so those two files look like a good starting point. I would aim at using the `URLPATH` environment variable such that the request goes to `example.com/URLPATH/...` . 

Thanks again for your time!
